products/almalinux.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: /almalinux
+layout: post
+title: AlmaLinux
+category: os
+command: lsb_release --release
+releasePolicyLink: https://blog.cloudlinux.com/announcing-open-sourced-community-driven-rhel-fork-by-cloudlinux
+activeSupportColumn: true
+releaseDateColumn: true
+sortReleasesBy: 'release'
+iconSlug: NA
+releases:
+  - releaseCycle: "8"
+    release: 2029-03-01
+    support: true
+    eol: false
+    latest: "8"
```

Comment:
Should this be 8.5?

https://wiki.almalinux.org/release-notes/8.5.html#providing-feedback-and-getting-help

Reply:
Yes good catch